### DISCUSSION TITLE
Welcome to image-processing Discussions!

### DISCUSSION THREAD

**Discussion by yassinfouzai** [yassinfouzai/image-processing#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kurowyn** [comment]
We should rename the project to something cooler, like "proki" (a play on words on "image processing", which can be rewritten as "processing image", then abridged to "proci", in turn making it "proki").
